src/main/steps/applicant1/hub-page/right-column/content.ts

```diff
@@ -4,29 +4,42 @@ import { CommonContent } from '../../../common/common.content';
 import { APPLICANT_2, CHECK_CONTACT_DETAILS, RESPONDENT } from '../../../urls';
 
 const en = ({ isDivorce, isApplicant2, userCase }: CommonContent) => ({
-  applicationDownloadLink: `<a class="govuk-link" href="/downloads/${
-    isDivorce ? 'divorce-application' : 'application-to-end-civil-partnership'
-  }"
-  download="${isDivorce ? 'Divorce-application' : 'Civil-partnership-application'}">View the ${
-    isDivorce ? 'divorce application' : 'application to end your civil partnership'
-  } (PDF)</a>`,
-  certificateOfServiceDownloadLink:
-    '<a class="govuk-link" href="/downloads/certificate-of-service" download="Certificate-of-service">View your ‘certificate of service’ (PDF)</a>',
-  respondentAnswersDownloadLink: `<a class="govuk-link" href="/downloads/respondent-answers"
-  download="Respondent-answers">View the response to the ${
-    isDivorce ? 'divorce application' : 'application to end your civil partnership'
-  } (PDF)</a>`,
-  deemedOrDispensedDownloadLink: `View the court order granting your application for
-  ${
-    userCase.alternativeServiceOutcomes?.[0].value.alternativeServiceType === AlternativeServiceType.DISPENSED
-      ? 'dispensed'
-      : 'deemed'
-  } service (PDF)`,
-  deemedOrDispensedDownloadReference: `/downloads/${
-    userCase.alternativeServiceOutcomes?.[0].value.alternativeServiceType === AlternativeServiceType.DISPENSED
-      ? 'certificate-of-dispense-with-service'
-      : 'certificate-of-deemed-as-service'
-  }`,
+  applicationDownload: {
+    name: 'Divorce-Application',
+    reference: `/downloads/${isDivorce ? 'divorce-application' : 'application-to-end-civil-partnership'}`,
+    link: `View the ${isDivorce ? 'divorce application' : 'application to end your civil partnership'} (PDF)`,
+  },
+  certificateOfServiceDownload: {
+    name: 'Certificate-of-Service',
+    reference: '/downloads/certificate-of-service',
+    link: "View your 'certificate of service' (PDF)",
+  },
+  respondentAnswersDownload: {
+    reference: '/downloads/respondent-answers',
+    name: 'Respondent-Answers',
+    link: `View the response to the ${
+      isDivorce ? 'divorce application' : 'application to end your civil partnership'
+    } (PDF)`,
+  },
+  deemedOrDispensedDownload: {
+    reference: `/downloads/${
+      userCase.alternativeServiceOutcomes?.[0].value.alternativeServiceType === AlternativeServiceType.DISPENSED
+        ? 'certificate-of-dispense-with-service'
+        : 'certificate-of-deemed-as-service'
+    }`,
+    name: 'Certificate-of-Service',
+    link: `View the court order granting your application for
+    ${
+      userCase.alternativeServiceOutcomes?.[0].value.alternativeServiceType === AlternativeServiceType.DISPENSED
+        ? 'dispensed'
+        : 'deemed'
+    } service (PDF)`,
+  },
+  certificateOfEntitlementDownload: {
+    reference: '/downloads/certificate-of-service',
+    name: 'Certificate-of-Entitlement',
+    link: 'View the certificate of entitlement (PDF)',
+  },
```

Comment:
was there a specific reason for this? i feel like it became unnecessarily long

Reply:
I remember an older pr where Taleb commented it would be better to move the html into the template, so I was updating them all to follow suit. It's longer but cleaner imo

Reply:
i think he was talking about when we had massive html in one line but anchor tags were fine. Tbh i thought the other way was cleaner/easy to read but i understand your opinion as well that i dont mind. might be worth getting another eye on it.

Reply:
likewise, I'm happy either way but better to be consistent with it

Reply:
I think it's much cleaner this way. now it's all data/variables here, which then gets used in the template to create the markup.